digdag-standards/src/main/java/io/digdag/standards/command/ecs/EcsClientConfig.java

```diff
@@ -101,15 +107,17 @@ private static EcsClientConfig buildEcsClientConfig(String clusterName, Config e
                 .withSubnets(ecsConfig.getOptional("subnets", String.class))
                 .withMaxRetries(ecsConfig.get("max_retries", int.class, DEFAULT_MAX_RETRIES))
                 .withCapacityProviderName(ecsConfig.getOptional("capacity_provider_name", String.class))
-                .withCpu(ecsConfig.getOptional("cpu", Integer.class))
-                .withMemory(ecsConfig.getOptional("memory", Integer.class))
+                .withContainerCpu(ecsConfig.getOptional("container_cpu", Integer.class))
+                .withContainerMemory(ecsConfig.getOptional("memory", Integer.class))
```

Comment:
`container_memory` for consistent naming.

Reply:
Thanks. fixed 87b81b4